adal/build.gradle

```diff
@@ -201,4 +214,8 @@ task checkstyle(type: Checkstyle) {
     classpath = files()
 }
 
-assembleDebug.dependsOn 'checkstyle', 'pmd', 'findbugs', 'lint', 'createPom'
\ No newline at end of file
+tasks.whenTaskAdded { task ->
+    if (task.name == 'assembleDebug') {
+        task.dependsOn 'checkstyle', 'pmd', 'findbugs', 'lint', 'createPom'
+    }
```

Comment:
I'm not making it clear i think, sorry about that. I don't think we should skip the assembleRelease. If somebody else send us a PR and they run against release, are we going to allow them check in code doesn't pass static analysis?

Since you're changing the task, i would like us do the right thing.